lib/tapioca/config.rb

```diff
@@ -29,6 +29,7 @@ def outpath
     TAPIOCA_CONFIG = T.let("#{TAPIOCA_PATH}/config.yml", String)
 
     DEFAULT_COMMAND = T.let("bin/tapioca", String)
+    DEFAULT_COMPILER_PATH = T.let("#{__dir__}/compilers/dsl", String)
```

Comment:
As mentioned in a previous PR, this is a static path that never changes, and was being dynamically generated in the `Generator` class. Moved it here as it makes more sense and won't be affected by the file moving to a different namespace.

Reply:
I'm not sure about this one:
* Having it under `Config` implies this is something configurable.
* Having `default` in the name implies this is a default value that may be changed.
* It doesn't say anything about `dsl`

What do you think about moving it under `Tapioca::Compilers::Dsl::COMPILERS_PATH`?